### PR TITLE
Bump outdated jekyll-* dependencies

### PR DESCRIPTION
**Pull request by benbalter**
### Jekyll Feed

* fix <entry> template for posts with post.lang defined 

### Jekyll Relative links

* Support for relative static files
* Save from 'invalid byte sequence' errors

### Jekyll README index

* Support for READMEs in subfolders
* README detection now respects global Markdown extension setting

### CODE REVIEW

**Comment by parkr** [comment]
We don't need to lock the source for coffeescript anymore near as I can tell. Worth skipping this version of jekyll-coffeescript and going back to it being unlocked?

File: lib/github-pages/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module GitHubPages
-  VERSION = 130
+  VERSION = 131
```
Comment (parkr) [inline lib/github-pages/version.rb]: I'd rather see this happen automatically!

Comment (benbalter) [inline lib/github-pages/version.rb]: > I'd rather see this happen automatically!

How?!?!

Comment (benbalter) [inline lib/github-pages/version.rb]: Ohhhhhh

**Pull request merged**